lib/stones-spec/language/gobstones.rb

```diff
@@ -28,6 +28,10 @@ def self.is_runtime_error?(result)
       result.include? 'Error en tiempo de ejecución'
     end
 
+    def self.is_syntax_error?(result)
+      result.include? 'Error de sintaxis' or result.include? 'Error en el programa'
```

Comment:
You can move this to a `include_any?` message in the `Array` class, take a look at this: http://rubylearning.com/satishtalim/ruby_open_classes.html